packages/rn-tester/e2e/test/runners/buttonComponentScreenRunner.js

```diff
@@ -0,0 +1,41 @@
+import { defineFeature, loadFeature } from 'jest-cucumber';
+import { givenUserOnMainPage } from '../common_steps/commonSteps.steps';
+import { thenVerifyAlertBoxHasText, thenVerifyThatTheButtonComponentIsDisplayed, thenVerifyThatTheButtonHeaderIsDisplayed, whenUserClicksOnTheButtonComponent, whenUserClicksOnTheCancelApplicationButton, whenUserClicksOnTheOKButton, whenUserClicksOnTheSubmitApplicationButton } from '../steps/buttonComponentScreen.steps';
```

Comment:
This seems like unnecessary decoupling of code that's tightly coupled with this file. I believe we could move all of those helpers into this file and achieve similar readability and higher code colocation

Reply:
It has been resolved by importing all functions (*). Then one line to be able to use the functions without an alias. Please, check if the current implementation is suitable